lldb/source/Plugins/Process/gdb-remote/ProcessGDBRemote.cpp

```diff
@@ -876,7 +876,11 @@ bool ProcessGDBRemote::GPUBreakpointHit(void *baton,
       if (bp_sp) 
         bp_sp->SetEnabled(false);
     }
-    HandleGPUActions(response->actions);
+    if (Status err = HandleGPUActions(response->actions); err.Fail()) {
```

Comment:
TIL about "if with initializer syntax" that was added in c++17. Cool feature and keeps the error nicely scoped.

Reply:
You might consider simplifying this to:
```
Status err = HandleGPUActions(response->actions); 
if (err.Fail()) {
```
for clarity